plugins/_template.md

```diff
@@ -0,0 +1,9 @@
+# {{Plugin's name}}
+
+{{Plugin's description}}
```

Comment:
In description you include list of features ?

Reply:
You can put whatever you want ^^.